pytorch_toolkit/object_detection/tests/run_nncf_tests.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+""" This module contains unit tests. """
+
+import argparse
+import logging
+import os
+import sys
+import unittest
+
+from common.utils import run_tests_by_pattern
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--pattern', default='nncf_tests_*.py')
+    parser.add_argument('--verbose', action='store_true')
+
+    return parser.parse_args()
+
+
+def main():
+    if os.path.abspath(os.getcwd()) == os.path.abspath(os.path.join(os.path.dirname(__file__), '..')):
+        return 0
+
+    logging.basicConfig(level=logging.INFO)
+    args = parse_args()
+    # TODO: rewrite in a similar way tests for train and export
```

Comment:
Ilya, I think it is a good idea to rewrite other tests in such a way, but I did not spent time on it (it may be done later).

Should I do it right now?

Reply:
why not?

Reply:
Done. All the tests scripts `{domain_folder}/tests/run_{train|export|nncf}_tests.py` are rewritten in the similar way
(both for `object_detection` and `action_recognition_2` domains).

Now the scripts just call one function `run_tests_by_pattern`.

Please, note that now `--verbose` is still `store_true` argument, since it is converted into concrete unittest verbosity value inside the function `run_tests_by_pattern`.